src/test/java/com/ordestiny/tdd/designpattern/factorypattern/AnimalFactoryTest.java

```diff
@@ -0,0 +1,77 @@
+package com.ordestiny.tdd.designpattern.factorypattern;
+
+import com.ordestiny.tdd.designpattern.factorypattern.animal.*;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * @author Rachit Parikh
+ */
+
+public class AnimalFactoryTest {
+
+    Animal animal;
+
+    @Test
+    public void checkDog() throws Exception{

```

Comment:
For the test case methods naming, it is a good practice to state what you are going to test and what is your expected result. So others dont need to read your code in detail.

Reply:
I have added comments in the new PR.